cypress/e2e/navigation.cy.js

```diff
@@ -0,0 +1,7 @@
+describe('Navigation tests', () => {
+  it('Navigate to the home page and verify url', () => {
+    cy.visit('/');
+    cy.url().should('eq', Cypress.env('baseUrl'));
```

Comment:
If you want use config env. you should use Cypress.config('baseUrl'); instead of Cypress.env('baseUrl')